services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/StaticTableSchema.java

```diff
@@ -274,6 +281,19 @@ private Builder(Class<T> itemClass) {
             return this;
         }
 
+        /**
+         * Adds an {@link AttributeConverterProvider} to the table schema. The {@link AttributeConverterProvider} must
+         * provide {@link AttributeConverter}s for all types used in the schema.
```

Comment:
Does it add it to the table schema? It seems like it provides a higher-precedence alternative to the table schema.

Reply:
Let's go with that terminology, clearer.